src/IssueViz.Security.UnitTests/Taint/TaintList/TaintIssuesControlViewModelTests.cs

```diff
@@ -127,16 +123,14 @@ public void Ctor_NoActiveDocument_NoIssuesDisplayed()
         {
             var storeCollection = new ObservableCollection<IAnalysisIssueVisualization> { CreateIssueViz() };
 
-            var activeDocumentLocator = new Mock<IActiveDocumentLocator>();
-            activeDocumentLocator.Setup(x => x.FindActiveDocument()).Returns((ITextDocument) null);
+            var locator = CreateLocatorAndSetActiveDocument(null);
```

Comment:
Hopefully it's clear enough what this call means. If not we can modify it again.